src/components/Message/Message.story.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import { action } from '@storybook/addon-actions';
+
+import withTests from '../../util/withTests';
+import Message, { MessageIcon, MessageButton } from '.';
+import Heading from '../Heading';
+import Text from '../Text';
+import Button from '../Button';
+import ThumbsUpIcon from './thumbs-up.svg';
+
+storiesOf('Message', module)
+  .addDecorator(withTests('Message'))
+  .add(
+    'Default Message',
+    withInfo()(() => (
+      <Message>
+        <MessageIcon>
+          <ThumbsUpIcon />
+        </MessageIcon>
+        <Heading size={Heading.KILO} element="h4" margin={false}>
+          Transaction successfully refunded
+        </Heading>
+      </Message>
+    ))
+  )
+  .add(
+    'Message with button',
+    withInfo()(() => (
+      <Message>
+        <MessageIcon>
+          <ThumbsUpIcon />
+        </MessageIcon>
+        <div>
+          <Heading element="h4" size={Heading.KILO} margin={false}>
+            New Feature — Intelligent Reporting
+          </Heading>
+          <Text margin={false}>
+            Get automatic insights into your business statistics with one click.
+          </Text>
+        </div>
```

Comment:
Is this div necessary? If yes, maybe there is a cool way we can make this not required. Seems like it adds some hidden usage contract.

Reply:
It is, otherwise the flexbox layout breaks. I've updated the prop types to only allow a single child node. 
Of course it would be nicer to make it not required, but I can't think of a good way right now.